packages/data-transformer/src/interfaces/data-normalizer.interface.ts

```diff
@@ -0,0 +1,17 @@
+import {DataNormalizerUniqueKey} from "../types/data-normalizer-unique-key.type";
+
+export interface DataNormalizer<T, R> {
```

Comment:
Don't we usually add the suffix Interface ? 
```suggestion
export interface DataNormalizerInterface<T, R> {
```